e2e/react-start/rsc-hmr/tests/rsc-hmr-css.spec.ts

```
@@ -0,0 +1,134 @@
+import { expect } from '@playwright/test'
+import { test } from '@tanstack/router-e2e-utils'
+import { readFile, writeFile } from 'node:fs/promises'
```

ESLint

[error] `node:fs/promises` import should occur before import of `@playwright/test`

(import/order)